src/Core/Domain/Product/Supplier/Command/SetProductSuppliersCommand.php

```diff
@@ -30,35 +30,42 @@
 
 use PrestaShop\PrestaShop\Core\Domain\Product\Supplier\ProductSupplier;
 use PrestaShop\PrestaShop\Core\Domain\Product\ValueObject\ProductId;
-use PrestaShop\PrestaShop\Core\Domain\Supplier\Exception\SupplierException;
 use PrestaShop\PrestaShop\Core\Domain\Supplier\ValueObject\SupplierId;
+use RuntimeException;
 
 /**
  * Updates product suppliers
  */
-class UpdateProductSuppliersCommand
+class SetProductSuppliersCommand
```

Comment:
BC Break

Reply:
No this command was introduced in 178, so it's a simple refacto